k8s/spinnaker/README.md

```diff
@@ -10,97 +10,112 @@ If you would like to learn more about Spinnaker, please visit the
 
 Popular open stacks on Kubernetes packaged by Google.
 
+## About Spinnaker for Google Cloud Platform
+
+This solution installs a single instance of Spinnaker into a GKE cluster in a
+production-ready configuration. The installation follows recommended practices
+for running Spinnaker on Google Cloud Platform, and is integrated with related
+Google Cloud services, such as [Cloud Build](https://cloud.google.com/cloud-build/).
```

Comment:
At the moment, we don't have a GCB integration. Once 1.14 is out, and we bump the referenced version to 1.14, we will.

Reply:
Actually, not sure my comment is accurate.

Reply:
I was originally referring to the GCB stage, but I'm sure we can pre-configure the GCB pub/sub account (we've done this for GCR).

Reply:
That's what I had in mind here too. There's non-zero things we can already do to to integrate Spinnaker with Cloud Build, such as enabling triggering from a Cloud Build job. If we're not currently doing *any* such setup, I can replace this with GCR.